### PR TITLE
Add receptorctl work cancel doc and doc lint make targets

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -11,11 +11,24 @@ BUILDDIR      = build
...
+.PHONY: doc8 help lint Makefile rstcheck server
+
+doc8:
+	doc8 --ignore D001 .
+
+# Documentation linters
+lint: doc8 linkcheck rstcheck
+
+rstcheck:
+	-rstcheck --recursive --warn-unknown-settings .
```
Comment (oraNod) [inline docs/Makefile:27]: I think we should also add `doc8` and `rstcheck` to `docs/requirements.txt`. Older versions of rstcheck don't include the `--warn-unknown-settings` option so we probably want to pin that dependency.

```
$ which rstcheck
~/.local/bin/rstcheck

$ rstcheck --version
rstcheck 5.0.0

$ make rstcheck
rstcheck --recursive --warn-unknown-settings .
usage: rstcheck [-h] [--config CONFIG] [-r] [--report level] [--ignore-language language] [--ignore-messages messages]
                [--ignore-directives directives] [--ignore-substitutions substitutions] [--ignore-roles roles] [--debug] [--version]
                files [files ...]
rstcheck: error: unrecognized arguments: --warn-unknown-settings
```

However....

```
$ which rstcheck
~/git/receptor/venv/bin/rstcheck

$ rstcheck --version
rstcheck CLI Version: 6.2.1
rstcheck-core Version: 1.2.1
```

Using rstcheck installed in my venv works as expected.

File: docs/source/receptorctl/receptorctl_work_cancel.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+-----------
+work cancel
+-----------
+
+.. contents::
+   :local:
+
+``receptorctl work cancel`` cancels (kills) one or more units of work.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_cancel.rst]:
```suggestion
``receptorctl work cancel`` kills one or more units of work.
```

Comment (oraNod) [reply docs/source/receptorctl/receptorctl_work_cancel.rst]: I'd say it's better not to repeat the word "cancel" here. Consider "stops" or "terminates" as an alternative to "kills" - but I think it's fine given there is the standard `kill` command.

In the work unit docs the `canceled` state uses the word "terminate": https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_work_list.rst#work-unit-states

File: docs/source/receptorctl/receptorctl_work_cancel.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``Unit ID`` is a unique identifier for a work unit (job).  This should be the local Unit ID for the Receptor instance you are connected to,
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_cancel.rst]:
```suggestion
``Unit ID`` is a unique identifier for a work unit (job). When running the ``work cancel`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```

File: docs/source/receptorctl/receptorctl_work_cancel.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work cancel``command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_work_cancel.rst]:
```suggestion
``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work cancel`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.
```

**Review by oraNod — APPROVED** [review]

**Review by resoluteCoder — APPROVED** [review]

**Pull request merged**